client/src/app/users/user.component.html

```diff
@@ -1,11 +1,12 @@
-<mat-expansion-panel *ngIf="user" class="user-panel" matTooltip="email: {{user.email}}" #elem [id]="user.email">
+<mat-expansion-panel *ngIf="user" class="user-panel" #elem [id]="user.email">
 
-  <mat-expansion-panel-header>
+  <mat-expansion-panel-header  matTooltip="Expand {{user.name}}'s Profile">
```

Comment:
I would suggest that we should not have a tooltip here at all. We might want to use it elsewhere but it is often overused in 3601 projects. This expansion panel is probably an example of somewhere a tooltip isn't needed (and also they don't even appear for mobile users).

Reply:
The presence of tooltips is probably my fault, and some TAs from long, long ago. I suspect that @floogulinc is right, though, given the emphasis on mobile and the fact that tool tips really don't work well on mobile (if at all), and we should probably just step away from them.

Reply:
It seems from our conversations that we should remove the tooltips here before merging.